app/views/users/show.html.erb

```diff
@@ -26,7 +26,7 @@
       <%= podcast.name %>
     <h5>Description</h5>
       <%= podcast.description %>
-    <%= button_to "Approve", '/dashboard' %>
+    <%= button_to "Approve", "/approve_podcast/#{podcast.id}" %>
```

Comment:
This is what I was talking about earlier. Tried to make a path helper method but i think the button_to was jacking it up.

Reply:
Maybe giving it the method would help?